integreat_compass/core/settings.py

```diff
@@ -309,3 +309,59 @@
         },
     },
 }
+
+##########
+# EMAILS #
+##########
+
+#: The backend to use for sending emails (see :setting:`django:EMAIL_BACKEND` and :doc:`django:topics/email`)
+EMAIL_BACKEND = (
+    "django.core.mail.backends.console.EmailBackend"
+    if DEBUG
+    else "django.core.mail.backends.smtp.EmailBackend"
+)
+
+#: Default email address to use for various automated correspondence from the site manager(s)
+#: (see :setting:`django:DEFAULT_FROM_EMAIL`)
+DEFAULT_FROM_EMAIL = os.environ.get(
+    "INTEGREAT_COMPASS_SERVER_EMAIL", "keineantwort@integreat-compass.de"
+)
+
+#: The email address that error messages come from, such as those sent to :attr:`~integreat_compass.core.settings.ADMINS`.
+#: (see :setting:`django:SERVER_EMAIL`)
+SERVER_EMAIL = os.environ.get(
+    "INTEGREAT_COMPASS_SERVER_EMAIL", "keineantwort@integreat-compass.de"
+)
+
+#: A list of all the people who get code error notifications. When :attr:`~integreat_compass.core.settings.DEBUG` is ``False``,
+#: Django emails these people the details of exceptions raised in the request/response cycle.
+ADMINS = [("Integreat Compass Helpdesk", "tech@integreat-compass.de")]
+
+#: The host to use for sending email (see :setting:`django:EMAIL_HOST`)
+EMAIL_HOST = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST", "localhost")
+
+#: Password to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_HOST_PASSWORD`). If empty, Django won’t attempt authentication.
+EMAIL_HOST_PASSWORD = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST_PASSWORD")
+
+#: Username to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_HOST_USER`). If empty, Django won’t attempt authentication.
+EMAIL_HOST_USER = os.environ.get("INTEGREAT_COMPASS_EMAIL_HOST_USER", SERVER_EMAIL)
+
+#: Port to use for the SMTP server defined in :attr:`~integreat_compass.core.settings.EMAIL_HOST`
+#: (see :setting:`django:EMAIL_PORT`)
+EMAIL_PORT = int(os.environ.get("INTEGREAT_COMPASS_EMAIL_PORT", 587))
+
+#: Whether to use a TLS (secure) connection when talking to the SMTP server.
+#: This is used for explicit TLS connections, generally on port 587.
+#: (see :setting:`django:EMAIL_USE_TLS`)
+EMAIL_USE_TLS = bool(
+    strtobool(os.environ.get("INTEGREAT_COMPASS_EMAIL_USE_TLS", "True"))
+)
+
+#: Whether to use an implicit TLS (secure) connection when talking to the SMTP server.
+#: In most email documentation this type of TLS connection is referred to as SSL. It is generally used on port 465.
```

Comment:
@svenseeberg Apparently, port 465 seems to be deprecated, but does this mean SSL/TLS should not be used as well, or can it also be used via port 587? Or is STARTTLS via port 587 the only sensible option nowadays and SSL/TLS should be avoided altogether? In my mind, STARTTLS seems to be a bit inferior, since it sends the mail unencrypted if encryption fails?

Reply:
I think (most) servers abort the connection if STARTTTLS fails. But I agree that it seems better to encrypt the traffic to begin with.

Reply:
Oh interesting. https://datatracker.ietf.org/doc/html/rfc8314 and more interesting https://datatracker.ietf.org/doc/html/rfc8314#appendix-A. To sum things up: 465 seems to be the future ;)

Reply:
Interesting, thanks for the insight! Especially weird that a lot of these journalistic/blog-type explanation websites state that 465 is deprecated and 587 should be preferred, so probably they didn't do their research properly or just used outdated sources which predate the RFC you linked.

Back to the PR: Should we derive the default email port from the used method? Something along the lines of
```
if EMAIL_USE_TLS:
    EMAIL_DEFAULT_PORT = 587
elif EMAIL_USE_SSL:
    EMAIL_DEFAULT_PORT = 465
else:
    EMAIL_DEFAULT_PORT = 25

EMAIL_PORT = int(os.environ.get("INTEGREAT_COMPASS_EMAIL_PORT", EMAIL_DEFAULT_PORT))
```
?